autogpt_platform/backend/backend/executor/scheduler.py

```diff
@@ -168,6 +188,24 @@ async def _execute_graph(**kwargs):
         )
         # Clean up orphaned schedules for this graph
         await _cleanup_orphaned_schedules_for_graph(args.graph_id, args.user_id)
+    except GraphValidationError:
+        logger.error(
+            f"Scheduled Graph {args.graph_id} failed validation. Unscheduling graph"
+        )
```

Comment:
For the old schedule that doesn't have the IDs are we not going to notify? We can tell the name of it or something.

Reply:
no, as long as we log something graph_id is enough, don't we have user_id too? can we add that?